wgs/community-membership.md

```diff
@@ -139,8 +139,8 @@ The following apply to the part of codebase for which one would be an approver
 in an [OWNERS] file (for repos using the bot).
 
 - Reviewer of the codebase for at least 3 months
-- Primary reviewer for at least 10 substantial PRs to the codebase
-- Reviewed or merged at least 30 PRs to the codebase
+- Primary reviewer for at least 5 substantial PRs to the codebase
+- Reviewed or merged at least 10 PRs to the codebase
```

Comment:
This only mentioned reviewing PRs. I think we want to promote people who actually authored substantial PRs.

Reply:
+1. Similar comment as before. Combining to one, `Reviewer for or author of at least 10 substantial PRs to the codebase`

Reply:
+1 to Johnu's wording.

Reply:
ok, I updated to "Reviewer for or author of at least 5 substantial PRs to the codebase" in 2nd commit